roles/network/defaults/main.yml

```diff
@@ -38,6 +38,7 @@
 hostapd_wait: 5
 host_wireless_n: False
 driver_name: nl80211
+ap0_mac_addr: b8:27:99:12:34:56
```

Comment:
How was b8:27:99:12:34:56 chosen, just to shed light here for others?

Reply:
inspiration from https://github.com/iiab/iiab/pull/697/files